<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestao_colaboradores</class>
 <widget class="QDialog" name="fm_gestao_colaboradores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_fechar">
   <property name="geometry">
    <rect>
     <x>661</x>
     <y>483</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_colaboradores">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>145</y>
     <width>731</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>45</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Senha:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_senha"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>45</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Telefone:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_telefone">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>45</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Grupo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_grupo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>9</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_nome"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>46</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Usu√°rio:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_nome_de_usuario"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>82</y>
     <width>541</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="pushButton_salvar">
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_excluir">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ver_vendas">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ver vendas</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>112</y>
     <width>731</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Pesquisar:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pesquisar"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_pesquisar">
      <property name="text">
       <string>Perquisar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_nome</tabstop>
  <tabstop>lineEdit_nome_de_usuario</tabstop>
  <tabstop>lineEdit_senha</tabstop>
  <tabstop>lineEdit_telefone</tabstop>
  <tabstop>comboBox_grupo</tabstop>
  <tabstop>pushButton_salvar</tabstop>
  <tabstop>pushButton_excluir</tabstop>
  <tabstop>pushButton_cancelar</tabstop>
  <tabstop>pushButton_ver_vendas</tabstop>
  <tabstop>lineEdit_pesquisar</tabstop>
  <tabstop>pushButton_pesquisar</tabstop>
  <tabstop>tableWidget_colaboradores</tabstop>
  <tabstop>pushButton_fechar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
